src/helpers/emailHelper.js

```diff
@@ -0,0 +1,30 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+import EmailTemplates from './emailTemplates';
+
+dotenv.config();
+const { EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env;
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: EMAIL_ADDRESS,
+    pass: EMAIL_PASSWORD,
+  }
+});
+/**
+ * @export
+ * @class EmailService
+ */
+class emailHelper {
```

Comment:
Use pascal case for this class, ex: EmailHelper and remember its file to rename the same name as class